dev-scheme/racket/files/Makefile-nostrip.patch

```diff
@@ -0,0 +1,23 @@
+index 3393e0c55e..6a1c3e150b 100644
+--- a/bc/Makefile.in
++++ b/bc/Makefile.in
+@@ -505,8 +505,6 @@ unix-install-libs-cgc:
+ 	cd ..; $(ICP_LIB) bc/libmzgc.@LIBSFX@ "$(DESTDIR)$(libdir)/libmzgc.@LIBSFX@"
+ 	cd ..; $(ICP_LIB) bc/libracket.@LIBSFX@ "$(DESTDIR)$(libdir)/libracket.@LIBSFX@"
+ 	cd ..; $(ICP_STATIC_LIB) bc/rktio/librktio.@LIBSFX@ "$(DESTDIR)$(libdir)/librktio.@LIBSFX@"
+-	cd ..; $(STRIP_LIB_DEBUG) "$(DESTDIR)$(libdir)/libmzgc.@LIBSFX@"
+-	cd ..; $(STRIP_LIB_DEBUG) "$(DESTDIR)$(libdir)/libracket.@LIBSFX@"
```

Comment:
I wonder about this patch. AFAICS the `--disable-strip` configure option already causes `STRIP_LIB_DEBUG` to be defined as a no-op (namely `:`), so why is this needed in addition?

Reply:
true, I removed this while searching for solution, forgot to remove from patch